cosmos/profiles/snowflake/user_pass.py

```diff
@@ -40,6 +40,8 @@ class SnowflakeUserPasswordProfileMapping(BaseProfileMapping):
         "warehouse": "extra.warehouse",
         "schema": "schema",
         "role": "extra.role",
+        "host": "extra.host",
+        "port": "extra.port",
```

Comment:
Should we add or update tests to prevent regression in the future?

Reply:
Ok, will take a closer look at the test setup in a bit. 👀 I'm assuming we are aiming more for a unit-style test here, and not a full integration test that actually spins up the local Snowflake emulator, runs a DAG, and connects to, is that right? (still need to look into the existing tests in a bit more detail, just to confirm up front..).

Reply:
yes unit should be fine

Reply:
Done @pankajastro - added a test here: https://github.com/astronomer/astronomer-cosmos/commit/8fa38ec89e955e95cdaca78e53095aa21b1c4b5d#diff-4f553db19b8236959f5b683e10556536eed449f3a89bbeae775fd05419eedcd3R236-R257 (still waiting for the CI build to finish.. 🔁 )